### PR TITLE
Add a schemas endpoint. Closes #44.

### LINKED ISSUE #44: Produce /schemas endpoint
So that developers can easily have access to the schema, for validation and such, we should give them a way to easily access the current schema via an api endpoint.

We should post one `/schemas` sub-endpoint for each of the top-level endpoints (`/team`, `/projects`, `/working-groups`, etc.)

cc: @jeremiak

### PR DESCRIPTION
**Pull request by ertzeid**

### CODE REVIEW

File: lib/team_api/api.rb [hunk 1]
```diff
@@ -58,5 +59,12 @@ def self.generate_tag_category_endpoints(impl)
     end
 
     private_class_method :generate_tag_category_endpoints
+
+    def self.team_api_schema_location
+      about_yml = Gem::Specification.find_by_name 'about_yml'
+      "#{about_yml.gem_dir}/lib/about_yml/schema.json"
```
Comment (ccostino) [inline lib/team_api/api.rb]: What happens in the case that the gem isn't found?  Will this method still return a string of some sort or throw an exception?

File: lib/team_api/api_impl.rb [hunk 2]
```diff
@@ -32,6 +33,15 @@ def generate_index_endpoint(endpoint, title, description, items)
       }
     end
 
+    def generate_schema_endpoint(schema_file_location)
+      return if schema_file_location.nil? || schema_file_location.empty?
```
Comment (ccostino) [inline lib/team_api/api_impl.rb]: If we trigger this return statement, what will happen to the URL?  Is it just not generated at all and therefore not available as an endpoint (in other words, would it even be available)?  If not, should we hide the endpoint in that situation so folks can't get to a broken URL?

**Comment by ccostino** [comment]
@ertzeid:  Awesome, thank you!  I just had a couple of questions after reviewing.

File: lib/team_api/api_impl.rb (hunk 2 above)
Comment (ertzeid) [inline lib/team_api/api_impl.rb]: It won't have a `schemas` link at all, so it shouldn't result in a broken URL. Still not great if someone is expecting to find a schema, but at least better than a 404 :)

File: lib/team_api/api.rb (hunk 1 above)
Comment (ertzeid) [inline lib/team_api/api.rb]: Ooh, looks like it'll still return a string. Let me fix that.

File: lib/team_api/api_impl.rb (hunk 2 above)
Comment (ccostino) [inline lib/team_api/api_impl.rb]: Okay perfect, that's what I was getting at. :-)

**Comment by ccostino** [comment]
Looks good, merging!

**Pull request merged**